### PR TITLE
Add api/categories CRUD operations and testing

### PR DESCRIPTION
**Pull request by khaledkzy**

### CODE REVIEW

**Comment by khaledkzy** [comment]
[change the language of the picker item in the setting page](https://trello.com/c/4aVZcn3S/254-change-the-language-of-the-picker-item-in-the-setting-page)

**Comment by khaledkzy** [comment]
[Add API endpoints for Cateogries](https://trello.com/c/PIXuA8Jw/265-add-api-endpoints-for-cateogries)

**Comment by khaledkzy** [comment]
[Add DB Client for Categories](https://trello.com/c/qd3RaN6t/263-add-db-client-for-categories)

File: server/routes/admin/categories/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const express = require('express')
```
Comment (kharazzoro) [inline server/routes/admin/categories/index.js]: You should move this file to routes/api/categories/index.js
You are working in api file so it should be in API routers not admin

**Pull request merged**